docs/proposals/20190610-machine-states-preboot-bootstrapping.md

```diff
@@ -160,19 +160,19 @@ type MachineStatus struct
     - **Version [optional]**
         - Type: `*string`
         - Description: Version specifies the current version of Kubernetes running on the corresponding Node. This is meant to be a means of bubbling up status from the Node to the Machine. It is entirely optional, but useful for end-user UX if it’s present.
-    - **Bootstrap [optional]**
-        - Type: `ControllerPhaseType`
-        - Description: The state of the bootstrap provider.
-    - **Infrastructure [optional]**
-        - Type: `ControllerPhaseType`
-        - Description: The state of the infrastructure provider.
+    - **BootstrapReady [optional]**
+        - Type: `*string`
```

Comment:
```suggestion
        - Type: `*bool`
```

Reply:
I'm still not convinced this should be a pointer

Reply:
Maybe it doesn't matter if it is optional or not for Status, since that isn't something the user would generally set.

Reply:
Should we open an issue to discuss the pros/cons?